src/util/deploy-global.js

```diff
@@ -2,33 +2,49 @@ const { REST } = require('@discordjs/rest');
 const { Routes } = require('discord-api-types/v9');
 const fs = require('fs');
 const path = require('path');
-const { botId, botAuth } = require('../const/keys').get();
+const { botId, botAuth, adminGuild } = require('../const/keys').get();
```

Comment:
Should I add activityRank support server to keys.js?

Reply:
I'd suggest that you do, yes. Maybe once it's released change the command permissions accordingly too